OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -82,17 +82,19 @@ public ModBrowserLogic(Widget widget)
 			{
 				try
 				{
-					using (var preview = new Bitmap(Platform.ResolvePath(ModMetadata.CandidateModPaths[mod.Id], "preview.png")))
-						if (preview.Width == 296 && preview.Height == 196)
-							previews.Add(mod.Id, sheetBuilder.Add(preview));
+					using (var stream = ModMetadata.AllMods[mod.Id].Package.GetStream("preview.png"))
```

Comment:
Note that this will temporarily break the ability to view these files for mods in the support dir, but, like the other cases, this will be restored in #10703.